lib/awful/mouse/init.lua

```diff
@@ -326,6 +325,19 @@ end
 
 function mouse.object.set_current_widget() end
 
+--- Get the current widget geometry.
+-- @property current_widget_geometry
+-- @tparam ?table The geometry.
+-- @see current_widget
+
+function mouse.object.get_current_widget_geometry()
+    local _, ret = mouse.object.get_current_widget()
+
+    return ret
+end
```

Comment:
This handles `get_current_widget`. What about `get_current_widgets` (plural)?
Perhaps it is easier to make `get_current_widgets`  just return the result from `:find_widgets` instead of extracting the individual widgets (and perhaps renaming it to add `_geometries`)? I don't know.